### PR TITLE
add vis for social nav + fix policy init issue

### PR DESCRIPTION
**Pull request by jimmytyyang**
- add visual as Eric suggested for using social nav
- See the video: https://drive.google.com/file/d/19Bn-_S7NjuK06iasnNnu39IOzp9J6nIs/view?usp=sharing
- Fix policy init issue as Ruslan suggested

### CODE REVIEW

File: examples/siro_sandbox/controllers/fetch_baselines_controller.py [hunk 1]
```diff
@@ -510,14 +510,14 @@ def act(self, obs, env):
                 "agent_0_oracle_nav_action"
             ]._path_to_point(human_pos)
             # Sanitize the policy input even thought we did not call that here
-            self._init_policy_input()
+            self._policy_info = self._init_policy_input()
```
Comment (rpartsey) [inline examples/siro_sandbox/controllers/fetch_baselines_controller.py:513]: Why do we sanitize (self._policy_info = self._init_policy_input()) the policy at every act call?

**Review by 0mdc — APPROVED** [review]
LGTM!

File: examples/siro_sandbox/controllers/fetch_baselines_controller.py [hunk 2]
```diff
@@ -430,7 +430,7 @@ def act(self, obs, env):
 
         if self.current_state == FetchState.WAIT:
             self.current_state = FetchState.FOLLOW
-            self._init_policy_input()
+            self._policy_info = self._init_policy_input()
```
Comment (eundersander) [inline examples/siro_sandbox/controllers/fetch_baselines_controller.py:433]: How about rename `_init_policy_input` to `_get_policy_info`?

**Review by eundersander — APPROVED** [review]
one rename request, otherwise LGTM!

File: examples/siro_sandbox/controllers/fetch_baselines_controller.py (hunk 2 above)
Comment (jimmytyyang) [reply examples/siro_sandbox/controllers/fetch_baselines_controller.py:433]: done!

**Comment by jimmytyyang** [comment]
@rpartsey please let me if you have some comments for this PR :) Thanks a lot!

**Review by rpartsey — APPROVED** [review]
LGTM, thanks, Jimmy!

**Pull request merged**